### ISSUE TITLE
rules behind the limited number of electrodes.tsv per session should be made more explicit

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/bids-specification#1481]
Unless I missed it, it seems that the rules limiting the number of eletrodes.tsv per session are not explicitly stated.

For example

> electrodes (positions) and coordinates (in which they are expressed) are required to remain constant in a session. If not, then the data must be represented over multiple sessions (which can be on the same day). Also, if multiple EEG systems are being used in succession on the same subject (e.g. with dry electrodes), it should be represented in multiple sessions. 

Linking to the original thread at the end of this post.

I may be wrong but this feels more then a simple "how to use bids" issue as it adds an extra constraints on the definition of session for EEG and iEEG (as mentioned in the original thread).

Another reason why this may needed is that currently the validator does not enforce this rule and allows many more entities than the spec permits (https://github.com/bids-standard/bids-validator/issues/1198).

The lack of clarity on this point can otherwise lead to user confusion (see https://github.com/bids-standard/bids-matlab/issues/542#issuecomment-1529921282)

--- 

electrodes (positions) and coordinates (in which they are expressed) are required to remain constant in a session. If not, then the data must be represented over multiple sessions (which can be on the same day). Also, if multiple EEG systems are being used in succession on the same subject (e.g. with dry electrodes), it should be represented in multiple sessions. 

Different sampling rates and/or filter settings of the amplifier can happen within a session, as (of course) multiple runs of the same task, or different tasks in different (EEG) files.

Electrode positions are (potentially noisy) measurements that can be done multiple times, hence the `acq` label. The coordinate system is a definition,  which only needs to be specified once. 

I also do not recall why there is an optional `run` label for electrodes. I do not see the use case for it. Looking in the draft [google doc](https://docs.google.com/document/d/1ArMZ9Y_quTKXC-jNXZksnedK2VHHoKP3HCeO5HPcgLE/edit#), I also don't see it there.

_Originally posted by @robertoostenveld in https://github.com/bids-standard/bids-specification/issues/416#issuecomment-587459530_

**Comment by Remi-Gau** [comment]
Just to be clear, I am not trying to argue against the rationale, but more arguing for the rationale to be made more explicit, a bit like in the spirit of some other recent PRs that either clarify and make things more explicit:
- https://github.com/bids-standard/bids-specification/pull/1392
- https://github.com/bids-standard/bids-specification/pull/1444/files

**Comment by robertoostenveld** [comment]
I value clarifications!

I think it will be useful to look ahead at the animal ephys and at the NIBS BEPs. They both involve "something" (electrodes and/or stimulation coils) being placed relative to/in the brain. When planning/discussing EEG we considered teh 80/20 principle and wanted to align with the most common measurements (which don't move the EEG cap in a session) and avoid spending time discussing (at that time more experimental) dry and wearable electrode caps. Hence we defined the one-electrodelocation-per-session rule to limit the scope and be able to finalize the BEP. This also aligned with iEEG recordings, using chronic/static electrode positions

With future BEPs we may want to loosen that requirement and engage the discussion that we postponed back then. 

Also for OPM MEG we will have to anticipate changes, although not in the electrodes.tsv, so perhaps not something to worry about here.
